examples/mnist/mnist-with-tqdm.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import print_function
```

Comment:
Maybe we can remove this file ?

Reply:
@rave78 here I meant to remove the file `examples/mnist/mnist-with-tqdm.py` and not the line 
`from __future__ import print_function`. But you are right we do not need the same line in `mnist.py` neither